steep_expectations.yml

```diff
@@ -0,0 +1,58 @@
+---
+- file: lib/ulid.rb
+  diagnostics:
+  - range:
+      start:
+        line: 128
+        character: 6
+      end:
+        line: 128
+        character: 64
+    severity: WARNING
+    message: The branch is unreachable because the condition is exhaustive
+    code: Ruby::ElseOnExhaustiveCase
+  - range:
+      start:
+        line: 233
+        character: 23
+      end:
+        line: 233
+        character: 27
+    severity: ERROR
+    message: Type `::Numeric` does not have method `to_i`
+    code: Ruby::NoMethod
```

Comment:
The code

https://github.com/kachick/ruby-ulid/blob/a06dfc99c42c5c9c2b476915d94e0c4440748f72/lib/ulid.rb#L233

I hope https://github.com/ruby/rbs/pull/1012 resolve this workaround.